lib/usd/ui/CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+#
+# =======================================================================
+# Copyright 2019 Autodesk, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =======================================================================
+#
+
+project(mayaUsdUI)
+
+if (POLICY CMP0074)
+    cmake_policy(SET CMP0074 OLD)
+endif()
```

Comment:
Still have a policy requirement here?

Reply:
Thanks Matt. I remember adding this a while ago to remove some cmake warnings. Since we've now updated cmake min version I'll check these.

Reply:
FYI, I looked into these policy statements and I don't think they are needed anymore. There were three places where we were setting 0074 to OLD. I removed all of them and didn't see any cmake warnings. I'll make sure these get removed in a future PR.